charts/tekton-pipelines/templates/common/tasks/kustomize.yaml

```diff
@@ -75,7 +75,7 @@ spec:
       workingDir: $(resources.inputs.kubernetes-repo.path)
       script: |
         export KUBEVAL_SCHEMA_LOCATION=https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master
-        kustomize build $(params.kustomize_overlay_path) | kubeval -f -
+        kustomize build $(params.kustomize_overlay_path) | kubeval --ignore-missing-schemas --kubernetes-version=1.24.0 -f -
```

Comment:
1. we can't hardcore the version of the kubernetes.
2. if you need version - then obtain it with a custom step or 
    2.1 pass it as param to the helm chart.

Reply:
This was a quick and dirty fix on Firday to make sure that current projects work (because otherwise otherwise validation works against latest version, which is 1.26 now, and some APIs are deprecated there).

I've fixd that, now task gets kubernetes version of current cluster.
![2023-02-20_20-30_V6wnGfI4](https://user-images.githubusercontent.com/3766033/220147110-cbcb1e82-cae6-43bf-aec9-606f6bbdfdd6.png)